.s2i/httpd-cfg/01-commercial.conf

```diff
@@ -181,6 +181,15 @@ AddType text/vtt                            vtt
     # Redirects for observability rework per https://github.com/openshift/openshift-docs/pull/71248
     RewriteRule ^container-platform/4.14/power_monitoring/(.*)$ /container-platform/4.14/observability/power_monitoring/$1 [NE,R=302,L]
 
+    # Redirects for observability reorg per mleonov
+    RewriteRule ^container-platform/(4\.12|4\.13|4\.14|4\.15)/distr_tracing/distr_tracing_rn/distr-tracing-rn-?(.*)$ /container-platform/$1/observability/distr_tracing/distr_tracing_rn/distr-tracing-rn-past-releases.html [NE,R=302]
```

Comment:
**thought:**  I checked the docs and this URLs `<version>/distr_tracing/...` does not exist in 4.11 to 4.15. 
- How will this rule run? 
- Will it run when someone is on 4.9 distr_tracing release notes, and changes to 4.15 using version picker?

**question:** ❔Do we need this one?
(Same comment for the `otel` one below)

Reply:
I believe the idea is that the files were moved into the /observability/ folder in https://github.com/openshift/openshift-docs/pull/73114, so if people have bookmarks or other permalinks into the content in its old location, these redirects will send them to the new location. @max-cx have I understood correctly?

Reply:
@bscott-rh, yes, there is one extra directory, `/observability/`.